debezium-event-api/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
The main reason is that we'll likely have at least three different maven modules that implement this: this new container engine, a Wildfly service for easily running and managing the container engine, and a client library for easily consuming events from Kafka. Defining this API as a separate module now will make it easier to do that later without incurring some of the implementation dependencies that the embedded module currently has.